packages/core/src/view/mixins/FoldingMixin.ts

```diff
@@ -92,7 +92,7 @@ type PartialFolding = Pick<
   | 'updateAlternateBounds'
 >;
 type PartialType = PartialGraph & PartialFolding;
-
+Client.imageBasePath = '/images'
```

Comment:
You are changing the configuration here and this impacts all users of the lib, not only the example.